release-tools/scripts/manifest.yml

```diff
@@ -28,7 +28,7 @@ urls:
     prod: s3://artifacts.opendistroforelasticsearch.amazon.com/
     #snapshots: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/
     releases: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/releases/
-    releases_final_build: 
+    releases_final_build: rc-build-712915909 
```

Comment:
There is a trailing space please remove it

Reply:
Change made.